build.gradle

```diff
@@ -41,7 +41,7 @@ android {
 
 dependencies {
     compileOnly 'androidx.legacy:legacy-support-v4:1.0.0'
-    api 'com.urbanairship.android:urbanairship-core:17.7.1'
+    api 'com.urbanairship.android:urbanairship-core:18.1.2'
```

Comment:
We should have a release today that fixes some issues with our IAA products, ideally you can hold off for that release

Reply:
sounds good, please let me know once your release is done. and I will address your other comments thanks!

Reply:
looks like the urban airship team has updated this to 18.1.3 - https://github.com/urbanairship/android-library/releases/tag/18.1.3

@rlepinski - I assume that @Mansi-mParticle can go ahead and complete this release once you give the thumbs up for her changes?

Reply:
Yes, 18.1.3

Reply:
updated to version 18.1.3 @rlepinski  could you please review the other changes too?